Header header
float64 speed_average        
float64 angle_forward   
float64 angle_rear